### PR TITLE
Add missing locale file for ga and add rake task to check for it

### PR DESCRIPTION
**Pull request by Gargron**

### CODE REVIEW

File: lib/tasks/repo.rake [hunk 1]
```diff
@@ -76,4 +76,19 @@ namespace :repo do
...
   end
+
+  task check_locales_files: :environment do
+    pastel = Pastel.new
+
+    missing_yaml_files = I18n.available_locales.reject { |locale| File.exist?(Rails.root.join('config', 'locales', "#{locale}.yml")) }
+    missing_json_files = I18n.available_locales.reject { |locale| File.exist?(Rails.root.join('app', 'javascript', 'mastodon', 'locales', "#{locale}.json")) }
+
+    if missing_json_files.empty? && missing_yaml_files.empty?
+      puts pastel.green("OK")
```
Comment (ykzts) [inline lib/tasks/repo.rake]:
```suggestion
      puts pastel.green('OK')
```

Prefer single-quoted strings when you don't need string interpolation or special symbols.

**Comment by Gargron** [comment]
I'm not sure why the new rake command is failing in CircleCI :confused:

**Comment by ykzts** [comment]
I think you need to add `- *install_system_dependencies` to `steps` of `check-i18n`.

**Comment by Gargron** [comment]
Oh, I wasn't aware it wasn't in the same environment as the other tests.

**Review by ykzts — APPROVED** [review]

**Pull request merged**